src/object/srv_obj_migrate.c

```diff
@@ -2612,12 +2613,17 @@ migrate_fini_one_ult(void *data)
 	ABT_mutex_unlock(tls->mpt_inflight_mutex);
 
 	migrate_pool_tls_put(tls); /* lookup */
-	ABT_eventual_wait(tls->mpt_done_eventual, NULL);
-
+	rc = ABT_eventual_wait(tls->mpt_done_eventual, NULL);
+	if (rc != ABT_SUCCESS) {
+		rc = dss_abterr2der(rc);
+		D_WARN("failed to migrate fini one ult "DF_UUID": "DF_RC"\n",
+		       DP_UUID(arg->pool_uuid), DP_RC(rc));
+	} else
+		rc = 0;
 	migrate_pool_tls_put(tls); /* destroy */
```

Comment:
[style] if you used { } in if, use "{}" for else is prefered even there is only one line.

Reply:
Thank you for letting me know! I will follow this convention in future.